### DISCUSSION TITLE
0.3.0

### DISCUSSION THREAD

**Discussion by srid** [srid/haskell-flake#164] [Announcements]
- #134: Add `autoWire` option to control generation of flake outputs
    - #138: Add `checks` to `outputs` submodule
    - #143: Changed `autoWire` to be an enum type, for granular controlling of which outputs to autowire.
- #137: Expose cabal executables as flake apps. Add a corresponding `outputs.apps` option, while the `outputs.localPackages` option is renamed to `outputs.packages` (it now contains package metadata, including packages and its executables).
  - #151: Use `lib.getBin` to get the bin output
- #148: Remove automatic hpack->cabal generation. Use `pre-commit-hooks.nix` instead.
- #149: Fix unnecessary re-runs of cabal2nix evaluation. Add a `debug` option to have haskell-flake produce diagnostic messages.
- #153: Add `config.defaults` submodule to allow overriding the default devShell tools added by haskell-flake

<hr /><em>This discussion was created from the release <a href='https://github.com/srid/haskell-flake/releases/tag/0.3.0'>0.3.0</a>.</em>

**Comment by herberteuler** [comment]
I arrived at this project by searching "nix haskell flake" on Google, and the link to this project was at the top of the results.
Compared to all other tools I have tested, this one is the only one that works seamlessly out of the box without any issues.
Thank you, @srid.

**Reply by srid** [reply]
Glad to know that, cheers!
